x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_connector_form.tsx

```diff
@@ -181,11 +179,12 @@ export const ActionConnectorForm = ({
           <EuiErrorBoundary>
             <Suspense
               fallback={
-                <EuiFlexGroup justifyContent="center">
-                  <EuiFlexItem grow={false}>
-                    <EuiLoadingSpinner size="m" />
-                  </EuiFlexItem>
-                </EuiFlexGroup>
+                <SectionLoading>
```

Comment:
I saw this `SectionLoading` component used once in the alert flyout and used it elsewhere to provide context as to what was being loaded and continuity so that loading the different sections felt the same.

Reply:
I noticed this change turns it from "m" to "xl".
I don't know what the standard is and what considerations should be taken in regards to size.

Might be worth checking the EUI guidelines..... 🤔 
But it might be a subjective thing... no idea 🤷

Reply:
I pinged the design team on #kibana-design since Michael is out until the end of the year and they said there are no hard spinner size guidelines. Using the `xl` instead of `m` here looked more consistent to me when I was manually testing.